src/bp-activity/bp-activity-filters.php

```diff
@@ -2927,3 +2930,40 @@ function bb_activity_delete_link_review_attachment( $activities ) {
 		}
 	}
 }
+
+/**
+ * Add activity description to activity media/document/video
+ *
+ * @param $media
+ */
+function bb_activity_update_description( $media ) {
```

Comment:
@milanhirapra Function name should be `bb_activity_update_media_video_document_description`

Reply:
@dipskakadiya I made this change